docs/table/construct_table.rst

```diff
@@ -471,6 +471,41 @@ To create a copy of an existing table that is empty (has no rows)::
  float64 float64
  ------- -------
 
+Empty array of a known size
+---------------------------
+
+If you do know the size your table will be, but don't know the values in
+advance, you can create a zeroed numpy array and build the astropy table from
+it::
+
+  >>> N = 4
+  >>> dtype = [('a', 'i4'), ('b', 'f8'), ('c', 'bool')]
+  >>> temp = np.zeros(N, dtype=dtype)
+  >>> t = Table(data=temp)
```

Comment:
I guess it's a matter of style, but to my eye it is more readable with just:
```
>>> t = Table(data=np.zeros(N, dtype=dtype))
```
But if you think it is better as two lines that's fine by me.